### ISSUE TITLE
PngExporterTests.ExportWithResolution not working on windows 8 or newer

### ISSUE THREAD

**Issue by benjaminrupp** [oxyplot/oxyplot#759]
The tests PngExporterTests.ExportWithResolution(2) and PngExporterTests.ExportWithResolution(4) are not working on new versions of windows cause the used font "Segoe UI" has changed with windows 8 (see https://en.wikipedia.org/wiki/Segoe#Segoe_UI).
I suggest changing the font of the chart of these tests. I think "Times New Roman" would be a good choise. At least this is working both on windows 7 and windows server 12.

**Comment by objorke** [comment]
I get different results for "Times New Roman" on Windows 10 (local) and Windows Server 2012 R2 (appveyor).
Should we include a font in the test assembly?
Maybe something else than Times New Roman? See license terms: https://www.microsoft.com/typography/Fonts/family.aspx?FID=9

**Comment by benjaminrupp** [comment]
You are right. A font in test assembly would be the best. This would solve the problem for all windows versions.
Unfortunately I am not able to access my developer pc in the momemt. 
Do you have any preferences regarding the font in test assembly?
I will make a new pull request as soon as I am back in office.

**Comment by objorke** [comment]
no preference. but it must have a compatible license

**Comment by objorke** [comment]
The tests are failing in #797.
Maybe the issue is font rendering and not the font itself?
